front/src/store/index.js

```diff
@@ -52,6 +53,31 @@ export default new Vuex.Store({
     },
     logout ({ commit }) {
       commit('CLEAR_USER_DATA');
+    },
+    recoveryPasswordSendEmail ({ commit }, payload) {
+      return new Promise((resolve, reject) => {
+        axios.post(`${serverPath}api/ForgotPassword`,
```

Comment:
please, change end of serverPath without '/' in config and put right all usage of server path after this